SwiftCoAP.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "SwiftCoAP",
+        "repositoryURL": "https://github.com/namiai/SwiftCoAP",
+        "state": {
+          "branch": "swift_pm",
```

Comment:
That's going to go away 😬 

Can we change it in the imports to a local package maybe? Since the examples are supposed to be near the code in any case.